### PR TITLE
Made named list full screen

### PR DESCRIPTION
**Pull request by reubenvarghese1**
Named list is now full screen based on community feedback (Currently still to be decided but in case we choose this route then we can merge this PR)

### CODE REVIEW

File: src/main/ui/app/component/NamedList.jsx [hunk 1]
```diff
@@ -27,16 +27,13 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
```
Comment (djazayeri) [inline src/main/ui/app/component/NamedList.jsx]: Please also delete this, as well as the "half" function above. These are no longer used, right?

Comment (reubenvarghese1) [inline src/main/ui/app/component/NamedList.jsx]: Sorry , don't know why I forgot about that. I've deleted that as well .

Comment (djazayeri) [inline src/main/ui/app/component/NamedList.jsx]: I meant also delete the line `const COLUMNS_IF_SIZE = 5;`

Comment (reubenvarghese1) [inline src/main/ui/app/component/NamedList.jsx]: Spare me if I still have missed something. Haven't really had much time due to the exams 😉

**Pull request merged**